vss-processor/src/main/kotlin/org/eclipse/kuksa/vssprocessor/VssDefinitionProcessor.kt

```diff
@@ -128,11 +130,50 @@ class VssDefinitionProcessor(
                     duplicateSpecificationNames,
                 )
 
-                val file = FileSpec.builder(PACKAGE_NAME, classSpec.name!!)
+                val className = classSpec.name ?: throw NoSuchFieldException("")
+                val fileSpecBuilder = FileSpec.builder(PACKAGE_NAME, className)
+
+                val parentImport = createParentImport(specModel, generatedFilesVssPathToClassName)
+                if (parentImport.isNotEmpty()) {
+                    fileSpecBuilder.addImport(PACKAGE_NAME, parentImport)
+                }
+
+                val file = fileSpecBuilder
                     .addType(classSpec)
                     .build()
 
                 file.writeTo(codeGenerator, false)
+                generatedFilesVssPathToClassName[vssPath.path] = className
+            }
+        }
+
+        // Uses a map of vssPaths to ClassNames which are validated if it contains a parent of the given specModel.
+        // If the actual parent is a sub class (Driver) in another class file (e.g. Vehicle) then this method returns
+        // a sub import e.g. "Vehicle.Driver". Otherwise just "Vehicle" is returned.
+        private fun createParentImport(
```

Comment:
Optional:
Somehow this name does not really fit. My personal expectation is, that when something is "created" it's normally a more complex object than a String. 
Maybe something like "findParent" or similar is more appropriate.

Reply:
Renamed to "build" like in StringBuilder.